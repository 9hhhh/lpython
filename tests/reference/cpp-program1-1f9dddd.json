{
    "basename": "cpp-program1-1f9dddd",
    "cmd": "lfortran --show-cpp {infile}",
    "infile": "tests/program1.f90",
    "infile_hash": "ec4ccdfed01d45648718ca57a9ceb3e5aa82e55e57d8906636902e7b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "cpp-program1-1f9dddd.stdout",
    "stdout_hash": "c7b831f293fe48995b1e60720c99b0cc105bc0391b117d27b9577e30",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}