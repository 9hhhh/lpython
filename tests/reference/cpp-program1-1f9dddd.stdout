#include <iostream>
#include <Kokkos_Core.hpp>
int main()
{
    int i;
    i = 5;
    std::cout << i << std::endl;
    std::cout << i + 1 << std::endl;
    return 0;
}
